packages/opentelemetry-metrics/src/export/Aggregator.ts

```diff
@@ -29,6 +29,19 @@ export class CounterSumAggregator implements Aggregator {
   }
 }
 
+/** Basic aggregator for Observer. */
+export class ObserverAggregator implements Aggregator {
+  private _current: number = 0;
+
+  update(value: number): void {
+    this._current = value;
+  }
+
+  value(): Sum {
```

Comment:
I think `Sum` is not right type here, Observer instruments use LastValue Aggregation (which is same as Gauge). I would add a new type `LastValue` (`export type LastValue  = number;`) and use it here.